src/lib/orm/migrations/Type600.ts

```diff
@@ -0,0 +1,91 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
```

Comment:
If the idea is that we run this migration on prod then it needs `IF NOT EXISTS` for sure for the table queries but other than that it's also far from complete because it's missing all the data migrations (INSERT INTO statements)